test/metric/query-json.json

```diff
@@ -1,23 +1,24 @@
-[
-  {
-    "Id": "m1",
-    "MetricStat": {
-      "Metric": {
-        "Namespace": "MetricValueBenchmarkTest",
-        "MetricName": "cpu_usage_active",
-        "Dimensions": [
-          {
-            "Name": "InstanceId",
-            "Value": "i-095d623fa10a192e3"
-          },
-          {
-            "Name": "cpu",
-            "Value": "cpu-total"
-          }
-        ]
-      },
-      "Period": 60,
-      "Stat": "Average"
+{
+  "MetricDataQueries":[
+    {
+      "Id": "m1",
+      "MetricStat": {
+        "Metric": {
+          "Namespace": "MetricValueBenchmarkTest",
```

Comment:
Did you mean to edit this? I think I actually added this intending to help people debug quickly as an example

Reply:
Yes. It would be easier for them to use 
```
aws cloudwatch get-metric-data --cli-input-json file://query-json.json
```

Reply:
But if you put the Start and End time here, this becomes stale.